src/Sulu/Bundle/TagBundle/Event/TagMergedEvent.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TagBundle\Event;
+
+use Sulu\Bundle\EventLogBundle\Event\DomainEvent;
+use Sulu\Bundle\TagBundle\Tag\TagInterface;
+
+class TagMergedEvent extends DomainEvent
+{
+    /**
+     * @var int
+     */
+    private $sourceTagId;
+
+    /**
+     * @var int
+     */
+    private $sourceTagName;
+
+    /**
+     * @var TagInterface
+     */
+    private $destinationTag;
+
+    public function __construct(
+        int $sourceTagId,
+        string $sourceTagName,
+        TagInterface $destinationTag
+    ) {
+        parent::__construct();
+
+        $this->sourceTagId = $sourceTagId;
+        $this->sourceTagName = $sourceTagName;
+        $this->destinationTag = $destinationTag;
+    }
+
+    public function getEventType(): string
+    {
+        return 'merged';
+    }
+
+    public function getEventPayload(): array
+    {
+        return [
+            'destinationTagId' => $this->destinationTag->getId(),
```

Comment:
the `payload` is not saved to the database, so this information is only available while the event is handled and cannot be retrieved later on. i am not sure if that is okay or if we should add an additional `getEventContext` getter that will be persisted to the database for this kind of data? 

on the other hand, if we do this, it will be confusing that there is a `getEventPayload` and a `getEventContext` 😕